tensorflow/java/src/gen/cc/ops/op_generator.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_JAVA_SRC_GEN_CC_OPS_OP_GENERATOR_H_
+#define TENSORFLOW_JAVA_SRC_GEN_CC_OPS_OP_GENERATOR_H_
+
+#include <string>
+
+namespace tensorflow {
+
+class OpGenerator {
+ public:
+  explicit OpGenerator(const std::string& ops_lib);
+  virtual ~OpGenerator();
+
+  int Run(const std::string& output_path);
+
+ private:
+  const std::string ops_lib;
+  std::string package_name;
```

Comment:
Sorry, that's the result of moving some stuff around without connecting all the dots at the end, I'll mark it `const`.